supportedBackends/google/batch/src/main/scala/cromwell/backend/google/batch/actors/BatchPollResultMonitorActor.scala

```diff
@@ -47,6 +47,9 @@ class BatchPollResultMonitorActor(pollMonitorParameters: PollMonitorParameters)
       case event if event.name == CallMetadataKeys.VmEndTime => event.offsetDateTime
     }
 
+  override def extractVmInfoFromRunState(pollStatus: RunStatus): Option[InstantiatedVmInfo] =
+    pollStatus.instantiatedVmInfo
```

Comment:
Is `instantiatedVmInfo` ever populated here? If so, what's populating it?

Reply:
Right now this is the only reference besides the symbol definition. As a scaffolding step that seems fine, maybe throw a comment on it if you intend to check in this tranche of changes as-is.